odxtools/multiplexer.py

```diff
@@ -54,14 +54,14 @@ class MultiplexerDefaultCase:
 
     short_name: str
     long_name: str
-    structure_ref: Optional[str] = None
+    structure_ref: Optional[OdxLinkRef] = None
 
     def __post_init__(self):
         self._structure: Optional[BasicStructure] = None
 
-    def _resolve_references(self, id_lookup: Dict[str, Any]) -> None:
+    def _resolve_references(self, odxlinks: OdxLinkDatabase) -> None:
         if self.structure_ref is not None:
-            self._structure = id_lookup.get(self.structure_ref)
+            self._structure = odxlinks.resolve(self.structure_ref)
             if self._structure is None:
                 logger.warning(
```

Comment:
This warning will not longer trigger (dead code)

Maybe add `if self.structure_ref` ?

Reply:
> Maybe add if self.structure_ref ?

isn't this already there? (I think that we can assume that links to objects which should get internalized are valid. agree?)